test/integration/targets/ec2_group/tasks/main.yml

```diff
@@ -9,7 +9,13 @@
 # - include: ../../setup_ec2/tasks/common.yml module_name: ec2_group
 
 - include: ./credential_tests.yml
-- block:
+- module_defaults:
```

Comment:
Will module_defaults be backported? Or should these tests be updated to use module_defaults after the cp has been done?

Reply:
module_defaults was in 2.7, so for backporting to 2.7 this is fine. For 2.6 we'll have to either change to not use module_defaults or not backport the integration test at all.